modular_azurepeak/code/game/objects/items/ritualcircles.dm

```diff
@@ -1024,30 +1022,43 @@ var/forgerites = list("Ritual of Blessed Reforgance")
 			var/target = input(user, "Choose a host") as null|anything in folksonrune
 			if(!target)
 				return
-			if(do_after(user, 50))
-				user.say("Motive force, oh, violence!!")
-				if(do_after(user, 50))
-					user.say("A gorgeous buffet of violence, for you, for you!!")
-					if(do_after(user, 50))
-						user.say("A slaughter awaits!!")
-						if(do_after(user, 50))
-							icon_state = "graggar_active"
-							user.apply_status_effect(/datum/status_effect/debuff/ritesexpended_high)
-							graggararmor(target)
-							spawn(120)
-								icon_state = "graggar_chalky" 
-
-/obj/structure/ritualcircle/graggar/proc/graggararmor(src)
-	var/onrune = range(0, loc)
-	var/list/possible_targets = list()
-	for(var/mob/living/carbon/human/persononrune in onrune)
-		possible_targets += persononrune
-	var/mob/living/carbon/human/target
-	if(possible_targets.len)
-		target = pick(possible_targets)
-	else
-		to_chat(usr, "No valid targets are standing on the rune! You must stand directly on the rune to receive Graggar's blessing.")
-		return
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("MOTIVE FORCE, OH VIOLENCE!!")
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("A GORGEOUS FEAST OF VIOLENCE, FOR YOU, FOR YOU!!")
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("A SLAUGHTER AWAITS!!") // see the numbers taste the violence
+			if(!do_after(user, 5 SECONDS))
+				return
+			icon_state = "graggar_active"
+			user.apply_status_effect(/datum/status_effect/debuff/ritesexpended_high)
+			graggararmor(target)
+			spawn(120)
+				icon_state = "graggar_chalky" 
+		if("War Ritual")
+			to_chat(user, span_userdanger("This rite will get me more tired than usual... I wonder, should I proceed?"))
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("BLOOD FOR THE WAR GOD, THE CIRCLE IS DRAWN!!")
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("LET NOBLE FLESH BE THE PRICE FOR THE HORDE!!")
+			if(!do_after(user, 5 SECONDS))
+				return
+			user.say("LET PORTALS OPEN, LET 'HIS' CHILDREN COME!!")
+			if(!do_after(user, 5 SECONDS))
+				return
+			icon_state = "graggar_active"
+			if(perform_warritual())
+				user.apply_status_effect(/datum/status_effect/debuff/ritesexpended_high)
+			else
+				to_chat(user, span_smallred("The ritual fails. A noble, member of the inquisition or a tennite churchling body must be in the center of the circle!"))
```

Comment:
```suggestion
				to_chat(user, span_smallred("The ritual fails. A body of noble blood - from the inquisition - or from the church must be on the circle!"))
```
there is no center it's just a circle and also better clarification on who we want